src/renderers/shared/ReactPerf.js

```diff
@@ -344,14 +372,18 @@ function getMeasurementsSummaryMap(measurements) {
 }
 
 function start() {
+  returnWarnIfDevFalse();
   ReactDebugTool.beginProfiling();
 }
 
 function stop() {
+  returnWarnIfDevFalse();
+  alreadyWarned = false;
```

Comment:
Fix other things but need some clarification here, you mean I should leave 
```
if (!__DEV__) {
  warnInProduction();
  return [];
}
```
only in `start()` and 'stop()' and remove in other places?